sc2ts/inference.py

```diff
@@ -222,6 +222,13 @@ def submission_date(self):
     def submission_delay(self):
         return (self.submission_date - self.date).days
 
+    def get_hmm_cost(self, num_mismatches):
+        # Note that Recombinants objects have total_cost.
+        # This bit of code is sort of repeated.
+        num_parents = len(np.unique([seg.parent for seg in self.path]))
```

Comment:
Why not ``len(self.path)  -1`` here? I don't see any reason to start thinking about unique parents at this point, it's just complexity.

Reply:
This is what I was wondering about in the discussion. I ended up just following `total_cost` in the `Recombinant` class. It looks like a function of the number of parents (`len(fwd.parents)`), but it is actually the number of copying nodes in the path.

Reply:
Updated.